bin/etcd-manage

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eo pipefail
+
+preStart() {
+  # pull ip address for current machine and set in environment
+  export ETCD_IP=$(hostname -i)
```

Comment:
Couple things with this review item...

- The existing implementation opens the same ports.
- It's a necessary evil that a public port be open, for the bootstrap node only, in order to `curl -X PUT` the cluster discovery metadata.
- I removed the public IP/ports from the actual bootstrapped cluster.

I believe this satisfies the existing requirement.